ble_controller/CHANGELOG.rst

```diff
@@ -21,6 +24,18 @@ Changes
 
 * When LLPM mode is enabled, the connection event length is now no longer implicitly set to 1 ms.
 
+Known issues and limitations
+============================
+
+* In connections, the Link Layer payload size is limited to 27 bytes on LE Coded PHY (DRGN-8476).
+* If the application adds an all zeroes IRK, it will be treated as a valid entry in the device identity list. An all zeroes IRK is invalid and must not be added (DRGN-9083).
+* The ble_controller will generate a resolvable address for the TargetA field in directed advertisements if the target device address is in the device identity list with a non-zero IRK, even if privacy is not enabled and the local device address is set to a public address. A workaround is to remove the device address from the device identity list (DRGN-10659).
```

Comment:
Since this is known issues for the BLE controller I think we should use LL terminology, device identity list -> resolving list.